js/render/painter.js

```diff
@@ -374,3 +377,20 @@ Painter.prototype.setExMatrix = function(exMatrix) {
 Painter.prototype.lineWidth = function(width) {
     this.gl.lineWidth(util.clamp(width, this.lineWidthRange[0], this.lineWidthRange[1]));
 };
+
+Painter.prototype.overdrawInspector = function(enabled) {
```

Comment:
I'm going to change this to `showOverdrawInspector` to match the external property.